ironfish-cli/src/commands/service/snapshot.ts

```diff
@@ -99,7 +110,36 @@ export default class CreateSnapshot extends IronfishCommand {
       const snapshotBaseName = path.basename(SNAPSHOT_FILE_NAME, '.tar.gz')
       const snapshotKeyName = `${snapshotBaseName}_${timestamp}.tar.gz`
 
-      const s3 = new S3Client({})
+      let s3 = new S3Client({})
+      if (flags.r2) {
+        const secret_name = 'r2-prod-access-key'
```

Comment:
```suggestion
        const secretName = 'r2-prod-access-key'
```

Should this be camelCase?